### DISCUSSION TITLE
Question about Hacktoberfest

### DISCUSSION THREAD

**Discussion by Thre4dripper** [MakeContributions/DSA#988] [Q&A] [answered]
I have given a PR in this repo (Made a BST validation algorithm). but I didn't have received any mail by hacktoberfest about my PR. So does this all work. and when will I receive it.

**Comment by ming-tsai** [comment, marked as answer]
We are not participating the event Hacktoberfest 2022
For more information please see the issue #968

**Comment by ming-tsai** [comment]
I will add the tag but I guess it is going to show `Exclusive` tag on hacktoberfest page
